CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add stream template to package json. [#335](https://github.com/elastic/integrations-registry/pull/335)
 * Add support for multiple inputs per dataset. [#346](https://github.com/elastic/integrations-registry/pull/346)
 * Add experimental releases for packages and datasets. [#382](https://github.com/elastic/integrations-registry/pull/382)
+* Mark existing, non-released packages as experimental. [#384](https://github.com/elastic/package-registry/pull/384)
```

Comment:
Nit: I would skip this in the changelog as the changelog should focus only on registry changes.

Reply:
Fixed!